Gopkg.toml

```diff
@@ -60,7 +60,7 @@ required = ["gopkg.in/fsnotify.v1"]
 
 [[constraint]]
   name = "github.com/Peripli/service-manager"
-  branch = "master"
+  branch = "async_phase1"
```

Comment:
should be reverted before merging

Reply:
Yes. I will wait for the _async_phase1_ branch to be merged, revert the _.toml_ and then merge this PR.